### DISCUSSION TITLE
user not found after bind singature data

### DISCUSSION THREAD

**Discussion by bondnavigator** [Chia-Network/chia-blockchain#4582] [Support]
when I'm use email @hotmail and cannot found data in Revenue that mining (can bind signature normally)
then I'm created email @gmail for solved above I'm cannot bind signature  and faced issue not found user as below
 
![noha](https://user-images.githubusercontent.com/84022152/117821300-a438f700-b295-11eb-8182-115b013685bd.PNG)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
